### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by btkelly**
Resolves #78

### CODE REVIEW

File: gandalf/build.gradle [hunk 1]
```diff
@@ -65,9 +65,9 @@ gnag {
...
 
-    compile 'com.android.support:appcompat-v7:23.2.0'
-    compile 'com.squareup.okhttp3:okhttp:3.0.1'
-    compile 'com.google.code.gson:gson:2.5'
+    compile 'com.android.support:appcompat-v7:25.2.0'
+    compile 'com.squareup.okhttp3:okhttp:3.6.0'
+    compile 'com.google.code.gson:gson:2.8.0'
 
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
```
Comment (stkent) [inline gandalf/build.gradle]: Isn't mockito well into the 2.x's now?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Mon Oct 24 18:05:17 EDT 2016
+#Mon Mar 06 11:35:42 EST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (stkent) [inline gradle/wrapper/gradle-wrapper.properties]: May as well update to 3.4 now?

Comment (btkelly) [inline gradle/wrapper/gradle-wrapper.properties]: I was going to leave it at the Android Studio recommended version but if you've used 3.4 without issue I can bump it.

File: gandalf/build.gradle (hunk 1 above)
Comment (btkelly) [inline gandalf/build.gradle]: No idea, I didn't even check the testCompile ones. I'll take a look.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (stkent) [inline gradle/wrapper/gradle-wrapper.properties]: Yup, 3.4 is working for me on multiple projects!

File: gandalf/build.gradle (hunk 1 above)
Comment (stkent) [inline gandalf/build.gradle]: 👍

Comment (stkent) [inline gandalf/build.gradle]: Don't you have the dependency updates plugin set up? Recommended if not. https://github.com/ben-manes/gradle-versions-plugin

**Comment by stkent** [comment]
Looks like travis YAML file needs updates also.

**Comment by stkent** [comment]
Now that GitHub has pull request reviews that can be used for blocking a build, is it time to remove pullapprove?

**Comment by btkelly** [comment]
PullApprove has been removed, the status check will remain for this PR though as it was already posted.

**Comment by btkelly** [comment]
There seems to be an [issue with Travis CI](https://github.com/travis-ci/travis-ci/issues/6040) not downloading tools newer than 23.0.1. I've included a workaround that is reported to have fixed the issue until Travis gets around to updating.

**Comment by stkent** [comment]
Yup, I've seen that workaround before. SMH.

**Comment by stkent** [comment]
LGTM!

**Pull request merged**